### PR TITLE
[WIP] Pin down ginkgo version

### LINKED ISSUE #3361: pin down the version of ginkgo on CI scripts
/kind user-story

## User Story
pin down the version of ginkgo on CI scripts so that an upgrade doesn't give us unexpected failures

## Links 
- Feature Request: 
- Related issue: https://github.com/openshift/odo/issues/3359

/kind user-story

### PR DESCRIPTION
**Pull request by amitkrout**
**What type of PR is this?**

/kind feature

**What does does this PR do / why we need it**:
Pin down ginkgo version to avoid any uncertain failure on latest ginkgo version
**Which issue(s) this PR fixes**:

Fixes #3361 

**PR acceptance criteria**:

- [ ] Unit test : NA

- [ ] Integration test : NA

- [ ] Documentation : NA

**How to test changes / Special notes to the reviewer**:
```make goget-tools``` should pin down the ginkgo version to 1.13.0

### CODE REVIEW

**Comment by mik-dass** [comment]
```
Receiving objects: 100% (5070/5070), 5.17 MiB | 21.44 MiB/s, done.
Resolving deltas: 100% (3270/3270), done.
+pushd /home/travis/gopath/src/github.com/onsi/ginkgo
./scripts/pin-ginkgo-version.sh: 17: ./scripts/pin-ginkgo-version.sh: pushd: not found
Makefile:104: recipe for target 'goget-tools' failed
make: *** [goget-tools] Error 12
```
All the tests are failing

File: scripts/pin-ginkgo-version.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+CHECKOUT_GINKGO_VERSION="v1.13.0"
```
Comment (kadel) [inline scripts/pin-ginkgo-version.sh:8]: the current version is 1.14.0 can we use that?

**Comment by amitkrout** [comment]
Will reopen once it is unblocked
/close

**Comment by kadel** [comment]
> Will reopen once it is unblocked

What is causing this to be blocked?

File: scripts/pin-ginkgo-version.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# Delete existing ginkgo repo
+if [ -d "$GOPATH/src/github.com/onsi/ginkgo" ]; then
+    rm -rf $GOPATH/src/github.com/onsi/ginkgo
+fi
+
+git clone https://github.com/onsi/ginkgo $GOPATH/src/github.com/onsi/ginkgo
+pushd $GOPATH/src/github.com/onsi/ginkgo
+git checkout $CHECKOUT_GINKGO_VERSION
+go install -mod=“” github.com/onsi/ginkgo/ginkgo
```
Comment (kadel) [inline scripts/pin-ginkgo-version.sh:18]: use normal double quotes and it should work 

```suggestion
go install -mod="" github.com/onsi/ginkgo/ginkgo
```

**Pull request closed without merging**